README.md

```diff
@@ -84,7 +84,7 @@ See http://bundler.io/compatibility for known issues.
 
 ### Contributing
 
-If you'd like to contribute to RubyGems, that's awesome, and we <3 you. Check out our [guide to contributing](https://github.com/rubygems/rubygems/blob/master/CONTRIBUTING.rdoc#how-to-contribute) for more information.
+If you'd like to contribute to RubyGems, that's awesome, and we <3 you. Check out our [guide to contributing](CONTRIBUTING.md) for more information.
```

Comment:
This is not related to `Removing support for dynamic API backend` maybe in another PR? :)

Reply:
Sure thing